Python/Environment_Variables/README.md

```diff
@@ -0,0 +1,14 @@
+# Find Environment Variables
+
```

Comment:
Can we have a more elaborate README, I think an image or 2 would be a great idea.
Author section too maybe?

Reply:
Elaborated the README. I'm not sure what are you expecting in the author's section.

Reply:
Looks good to me